### DISCUSSION TITLE
The more you take, the more you leave behind. What am I?

### DISCUSSION THREAD

**Discussion by meviru** [meviru/badge-hunting#9] [Galaxy Trivia] [answered]
Puzzle: The more you take, the more you leave behind. What am I?

**Comment by seelew517** [comment, marked as answer]
footsteps

**Reply by meviru** [reply]
Congratulations!! 👏👏 You've just earned the [Galaxy Brain](https://github.com/seelew517?tab=achievements&achievement=galaxy-brain) badge. Keep it up, and remember to give the repository a star to show your support! 🎉

**Reply by seelew517** [reply]
starred. thx
